packages/core/base/core-base-common/src/index.ts

```diff
@@ -9,3 +9,5 @@
 export type { PluginOpaqueId, PluginName, DiscoveredPlugin } from './plugins';
 export { PluginType } from './plugins';
 export { EUI_STYLES_GLOBAL } from './eui';
+export { ServiceStatusLevels } from './service_status';
```

Comment:
`core-base-common` seems to be morphing into a "catch-all" I-don't-know-where-to-put-this-thing package. It's fine for now as the package is still very small. During the cleanup phase or slightly prior to that, we should audit everything in `core-base-*` packages and see if we could organize the components into a domain-like structure.

Reply:
> core-base-common seems to be morphing into a "catch-all" I-don't-know-where-to-put-this-thing package

Yup, I totally agree. The intent is to avoid the 'lets create 99 another one-file packages before the end of the migration' effect. 

> During the cleanup phase or slightly prior to that, we should audit everything in core-base-* packages and see if we could organize the components into a domain-like structure

Fully agree too.

Reply:
From a quick search through `core`, it's `elasticsearch`, `status`, and `savedObjects` that all use the `ServiceStatusLevels`.  Am I missing any other `core` domains in this list?

Reply:
No, AFAIK for now only the ES and SO services declare their statuses.